website/docs/d/api_management.html.markdown

```diff
@@ -41,6 +41,8 @@ output "api_management_id" {
 
 * `gateway_regional_url` - The URL for the Gateway in the Default Region.
 
+* `identity` - (Optional) An `identity` block as defined below.
```

Comment:
By checking the code of data source `api_management`, it seems that it does not have an attribute of `identity`, could you please add this attribute to its schema?

Reply:
Thanks, good find! I think I've done it right in 2cdee05, but perhaps it will soon be time to move more attributes that are common between the data source and the resource to common functions.